samples/maps/demo/data-class-two-ranges/demo.css

```diff
@@ -10,3 +10,40 @@
     text-align: center;
     color: gray;
 }
+
+#popup-pie {
+    width: 290px;
+    height: 240px;
+    padding: 0.5em 1em;
+    cursor: default;
+}
+
+#annotation-header {
+    display: flex;
+    justify-content: space-between;
+    width: 290px;
+    align-items: center;
+    color: #333;
+    font-size: 16px;
```

Comment:
Please use em / percents / rem instead of pixels. It will be more flexible when zooming in browser (accessibility).

Reply:
sure